docs/source/torch.rst

```diff
@@ -362,6 +362,8 @@ Reduction Ops
     amin
     max
     min
+    fmax
```

Comment:
These should be in the binary ops near minimum and maximum

Reply:
Addressed, Thanks for correction!